20090430181251_axtu-authen-client.spec.diff

########################################
this patch has problem here, which leads to a complation failure
g_arrayEssentialOptionValue[nCompanyMax]
rather than add "WT", we can substitude "WW" with "WT" thus no error
this fix already applied to svn, without sending a new IP
########################################






Index: /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/conf/axtu-authen-client-addr.conf
===================================================================
--- /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/conf/axtu-authen-client-addr.conf	(revision 7811)
+++ /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/conf/axtu-authen-client-addr.conf	(working copy)
@@ -6,11 +6,12 @@
 ML = https://update.miraclelinux.com/auth/src/authentication.php
 RF = https://update.redflag-linux.com/auth/src/authentication.php
 VS = https://asianux.org.vn/auth/src/authentication.php
+WT = https://update.wtec.co.th/auth/src/authentication.php
 
 [SignUp]
 HS = http://www.asianux.co.kr/register.php
 ML = http://www.miraclelinux.com/user
 RF = http://www.redflag-linux.com/ppd/product_user/register1.php
 VS = http://asianux.org.vn
-
+WT = http://www.wtec.co.th
 ########################################################
Index: /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/libs/classConfigCtrl.h
===================================================================
--- /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/libs/classConfigCtrl.h	(revision 7811)
+++ /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/libs/classConfigCtrl.h	(working copy)
@@ -17,6 +17,7 @@
 //! enum variable for Company-name 
 typedef enum {
 	NONE = 0,
+	WT,
 	WW, 
 	AP,
 	RF,
@@ -57,7 +58,7 @@
 const string g_arrayTagName[nSectionMax]={"AuthenticationServer","SignUp","TsnLocal"};
 **********************************************/
 const string g_arrayTagName[nSectionMax]={"AuthenticationServer","SignUp"};
-const string g_arrayEssentialOptionValue[nCompanyMax]={"NONE","WW","AP","RF","ML","HS","VS"};
+const string g_arrayEssentialOptionValue[nCompanyMax]={"NONE","WT","WW","AP","RF","ML","HS","VS"};
 //const string g_strEssentialOptionInitValue = "NONE";
 
 /*!
Index: /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/libs/classChecker.cpp
===================================================================
--- /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/libs/classChecker.cpp	(revision 7811)
+++ /home/peter/sds/packages/axtu-authen-client/1.5/tarball01/axtu-authen-client/libs/classChecker.cpp	(working copy)
@@ -43,6 +43,8 @@
 				c=ML;
 			}else if(nCompanyCode == 5){
 				c=VS;
+			}else if(nCompanyCode == 6){
+                                c=WT;
 			}else{
 				c=NONE;
 			}
Index: /home/peter/sds/packages/axtu-authen-client/1.5/axtu-authen-client.spec
===================================================================
--- /home/peter/sds/packages/axtu-authen-client/1.5/axtu-authen-client.spec	(revision 7811)
+++ /home/peter/sds/packages/axtu-authen-client/1.5/axtu-authen-client.spec	(working copy)
@@ -16,7 +16,7 @@
 
 Name: axtu-authen-client
 Version: 1.5.1
-Release:1%{?dist}
+Release:2%{?dist}
 Summary: An Authentication Client for the AXTU(AsianuX Tsn Updater)
 Group: Utilities/System
 License: GPL/LGPL
@@ -113,6 +113,9 @@
 
 
 %changelog
+* Tue Apr 30 2009 Peter, Wang<peter.wang@asianux.com> 1.5.1-2%{?dist}
+- Bug fix #6036 - Added wtec support
+
 * Mon Jul 7 2008 JeongHun Kim<jhkim@haansoft.com>, Kazuo YAGI <kyagi@miraclelinux.com> 1.5.1-1%{?dist}
 - Bug fix #3617 - Unified the error message to show the same message with tui and gui. - Kazuo YAGI <kyagi@miraclelinux.com>
 - Bug fix #4163 - Undefined warning message when no enough space directory /var/cache/axtu/ - JeongHun Kim<jhkim@haansoft.com>


